scripts/temp.sh

```diff
@@ -0,0 +1 @@
+./gradlew :aws-analytics-pinpoint:connectedAndroidTest
```

Comment:
Whats the purpose of this script, can you add some comments in here to justify adding something to scripts folder?